core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -290,14 +292,37 @@ class LogCleaner(initialConfig: CleanerConfig,
     }
 
     /**
-     * Clean a log if there is a dirty log available, otherwise sleep for a bit
+     * Sleeps if no logs have been cleaned
```

Comment:
Yeah, initially I thought it was duplicated since there was a similar docstring in `clean()`. I guess that the functionality is the same, so better have it duplicated